### PR TITLE
Laikad: Cache orbit and nav data

### PR DESCRIPTION
**Pull request by gijskoning**
Using Params to save orbits and nav from astro dog.
The cache file is measured to be ~ 3.6MB using sys.getsizeof(json_cache) and will be updated every minute with mostly new nav data. 
Only if no orbits are valid anymore, they will be downloaded and cached. And the orbit prediction file contain at max 24 hours of future ephemeris, but because of the delay from the source in processing these files when downloaded they are valid for around 12-18 hours.

Old ephemeris data is overwritten when new data is added: https://github.com/commaai/laika/pull/91 - Merged

### CODE REVIEW

File: selfdrive/locationd/laikad.py [hunk 1]
```diff
@@ -20,15 +21,31 @@
...
     self.gnss_kf = GNSSKalman(GENERATED_DIR)
     self.orbit_p: Optional[Process] = None
     self.orbit_q = Queue()
+    self.load_cache()
+    self.last_save_t = None
+    self.last_fetch_orbits_t = None
+
+  def load_cache(self):
+    params = Params()
+    if params.check_key(EPHEMERIS_CACHE) and False: # todo fix get params
```
Comment (adeebshihadeh) [inline selfdrive/locationd/laikad.py:39]: `Params::checkKey` only tells you whether this is a valid params key. it doesn't really make sense to use it here

Comment (gijskoning) [reply selfdrive/locationd/laikad.py]: Thanks! Will remove it

**Comment by gijskoning** [comment]
@adeebshihadeh Do you think 4 MB every minute is okay to cache?

**Comment by adeebshihadeh** [comment]
I think that should be fine. You can also try caching on exit.

**Pull request merged**